### PR TITLE
Introducing [REPLACE]

### LINKED ISSUE #84: Ability to remove org/project during swap - MAPS
**What would you like to be added**:

The ability to do what the deprecated legacy mode can do but within the MAPS method.
Currently, I cannot see a way to reproduce what the legacy method can do, which is to transform say `docker.io/a/b/c/d:v1` to `quay.io/x/y/z/d:v1`

Maybe we could use a new reserved keyword (**[replace]** in the example below) and wildcard, e.g.:

```
default::
[replace]quay.io*::docker.io/x/y/z
```

Expectation: `quay.io/a/b/c/d:v1 -> docker.io/x/y/z/d:v1`

The following would be equivalent, but now would not replace `quay.io/a/d:v1`.

```
default::
[replace]quay.io/a/b/*::docker.io/x/y/z
```

**Why is this needed**:

Sometimes the whole repo path needs amending, this ability appears to only be possible using legacy mode or exact matching - which is not ideal for when dealing with a large number of images.

### PR DESCRIPTION
**Pull request by M4C4R**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation

/kind feature
> /kind release

**What this PR does / why we need it**:

Introduces further flexibility to the MAPS swapping method.

**Which issue(s) this PR fixes**:

Fixes #84 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
New reserved keyword [REPLACE] which allows for full image path replacement using patterns.
```

**Additional documentation e.g., usage docs, etc.**:

```docs
Updated README.md
```

### CODE REVIEW

File: app/imageswap/imageswap.py [hunk 1]
```diff
@@ -239,15 +243,18 @@ def build_swap_map(map_file):
                     app.logger.warning(f"Invalid map is specified. Incorrect syntax for map definition. Skipping map for line: {line}")
                 continue
             # Store processed line key/value pair in map
-            if key.startswith("[EXACT]"):
-                key = key[7:].lstrip()
+            if key.startswith(imageswap_exact_keyword):
+                key = key[len(imageswap_exact_keyword) :].lstrip()
```
Comment (phenixblue) [inline app/imageswap/imageswap.py:247]: Thanks for this. Great update!

**Review by phenixblue — APPROVED** [review]
Thanks for this.

/lgtm

**Comment by phenixblue** [comment]
@M4C4R I have some things in my personal life that are taking up a lot of my time I normally use to maintain this project. Your changes have been merged, but I may be a little slow to cut a new release. Please bare with me.

**Comment by M4C4R** [comment]
@phenixblue no worries, thanks for merging :)

**Pull request merged**